examples/cmake/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ project(hello_librealsense2)
 # Find librealsense2 installed package
 find_package(realsense2 REQUIRED)
 
-# Enable C++11
+# Enable C++14
```

Comment:
Leftovers? should be 11 right?

Reply:
I don't think it matters, each example specifies it on its own -- but I'll check, may be leftover

Reply:
Changed